test/jake-util/knexfile-esm-module/seeds/one.js

```diff
@@ -0,0 +1 @@
+export function seed(...args) {}
```

Comment:
Is this intended syntax? Why use `...args`?

Reply:
You are correct, there is no need for the `...args`  here.
While the syntax is valid, there is no need to allocate unused variables.
I think I put them there for self documenting  reasons,.. and I just forgot to remove them when I realized there it was no need to assert it received the correct parameters, as this only needs to be successfully imported. and it shouldn't test the seeding functionality it self.

I'll remove the dead code, it might take bit, I pruned mi docker images, still downloading :)